src/main/config/config_eeprom.c

```diff
@@ -158,6 +159,15 @@ uint16_t getEEPROMConfigSize(void)
     return eepromConfigSize;
 }
 
+size_t getEEPROMStorageSize(void)
+{
+#ifdef EEPROM_IN_RAM
+    return EEPROM_SIZE;
```

Comment:
I still don't understand why we have to introduce the `EEPROM_SIZE` construct just for `EEPROM_IN_RAM`, and not just define `__config_start` and `__config_end` suitably to avoid this special case

Reply:
EEPROM_SIZE is used by the SITL target (aka EEPROM_IN_FILE) too, not just for EEPROM_IN_RAM, later PR's also use EEPROM_SIZE the same way.